src/components/ListingPage/styles.ts

```diff
@@ -32,54 +20,77 @@ export const FiltersContainer = styled.div`
 `;
 
 export const PageContainer = styled.div`
-  height: 100vh;
+  height: 100svh;
   width: 100%;
   display: flex;
   flex-direction: column;
-  padding-top: 1rem;
+
+  /* customize scrollbar for listing pages only */
+  ::-webkit-scrollbar {
+    width: 16px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    position: relative;
+    width: 100%;
+    background: rgba(204, 204, 204, 0.75);
+    border: 5px solid white;
+    border-radius: 20px;
+
+    &:hover {
+      border: 4px solid white;
+      background: rgba(204, 204, 204, 0.9);
+    }
+  }
 `;
 
 export const ListingCount = styled(H4)`
   margin-bottom: 0.5rem;
 `;
 
-export const CardColumn = styled.div`
-  overflow-y: auto;
-  display: flex;
-  flex-direction: column;
-  column-gap: 2rem;
+export const CardColumnWrapper = styled.div`
+  height: 100%;
+  width: clamp(400px, 33vw, 500px);
```

Comment:
Additionally, when I click between cases, the width of CardColumn keeps changing. Is this a problem with the clamp? 
<img width="1183" alt="Screen Shot 2024-05-01 at 9 14 44 AM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/e46bb9d2-d066-4577-a450-399d61095ae0">
<img width="1172" alt="Screen Shot 2024-05-01 at 9 14 36 AM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/d3d6c1fa-b5f6-44a9-a3a1-172cd853bcd7">
<img width="1169" alt="Screen Shot 2024-05-01 at 9 14 30 AM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/fa63a281-6308-4a6f-baa8-b387c0022605">

Reply:
yep, this is an issue with it being flex I think. I changed it to grid, should fix it.